src/py/flwr/server/superlink/fleet/vce/backend/raybackend.py

```diff
@@ -55,7 +55,12 @@ def __init__(
         runtime_env = (
             self._configure_runtime_env(work_dir=work_dir) if work_dir else None
         )
-        if backend_config.get("silent", False):
+
+        if backend_config.get("mute", False):
```

Comment:
Can we use a better name maybe? like `mute_logging`, `disable_logging`. Just for better readability.

Reply:
makes total sense! I'll replace it with `mute_logging`